### PR TITLE
Helps to stop 504 errors

### PR DESCRIPTION
**Pull request by c9dd**
As I can tell at the moment this helps to stop 504 errors, but I will keep an eye on this and tweak if/when needed.

### CODE REVIEW

**Comment by tlovett1** [comment]
I need to test this. Also needs formatting fixes.

File: config/nginx/default.conf [hunk 1]
```diff
@@ -28,6 +28,27 @@ server {
 		fastcgi_pass phpfpm:9000;
 		fastcgi_index index.php;
 		fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+
+
+        fastcgi_connect_timeout                 10;
```
Comment (cmmarslender) [inline config/nginx/default.conf:33]: We have some similar settings to these in V2, with slightly different values. I'd prefer to match these up to V2 for consistency: https://github.com/10up/wp-local-docker-v2/blob/master/src/config/nginx/develop.conf#L50

The main one that changed was the fastcgi_read_timeout on that config. Are you having issues that were fixed with a send timeout change?

File: config/nginx/default.conf [hunk 2]
```diff
@@ -28,6 +28,27 @@ server {
...
+        fastcgi_read_timeout                    180;
+        fastcgi_buffer_size                     512k;
+        fastcgi_buffers                         4       256k;
+        fastcgi_busy_buffers_size               512k;
+        fastcgi_temp_file_write_size            512k;
+        fastcgi_intercept_errors                on;
+        fastcgi_keep_conn                       on;
+
+        # uncomment these for HTTPS usage
+        #fastcgi_param  HTTPS                   $https if_not_empty;
```
Comment (cmmarslender) [inline config/nginx/default.conf:44]: I think we can just leave this particular line uncomment always and It should work fine on the backend

File: config/nginx/default.conf [hunk 3]
```diff
@@ -28,6 +28,27 @@ server {
...
+        #fastcgi_param  SSL_PROTOCOL            $ssl_protocol if_not_empty;
+        #fastcgi_param  SSL_CIPHER              $ssl_cipher if_not_empty;
+        #fastcgi_param  SSL_SESSION_ID          $ssl_session_id if_not_empty;
+        #fastcgi_param  SSL_CLIENT_VERIFY       $ssl_client_verify if_not_empty;
+
+        #fastcgi_pass wordpress:9000;
```
Comment (cmmarslender) [inline config/nginx/default.conf:50]: There isn't a service called `wordpress` - We shouldn't need this line with the fastcgi_pass that is already in this block which passes to `phpfpm`

File: config/nginx/default.conf [hunk 4]
```diff
@@ -28,6 +28,27 @@ server {
...
+        fastcgi_buffer_size                     512k;
+        fastcgi_buffers                         4       256k;
+        fastcgi_busy_buffers_size               512k;
+        fastcgi_temp_file_write_size            512k;
+        fastcgi_intercept_errors                on;
+        fastcgi_keep_conn                       on;
+
+        # uncomment these for HTTPS usage
+        #fastcgi_param  HTTPS                   $https if_not_empty;
+        #fastcgi_param  SSL_PROTOCOL            $ssl_protocol if_not_empty;
```
Comment (cmmarslender) [inline config/nginx/default.conf:45]: I don't think these 4 lines are required to be passed to `phpfpm` - As far as I'm aware, WordPress just looks and checks if `https` is on when doing anything ssl related - See function declaration here: https://developer.wordpress.org/reference/functions/is_ssl/

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Comment by jeffpaul** [comment]
@c9dd mind reviewing and responding to the feedback on the PR?

**Comment by jeffpaul** [comment]
@c9dd are you able to review and respond to the feedback on this PR?

**Comment by jeffpaul** [comment]
Resolves #29.

**Comment by jeffpaul** [comment]
@c9dd note that we're archiving WP Local Docker in favor of the newer [WP Local Docker v2](https://github.com/10up/wp-local-docker-v2), so please update to the v2 version and we can discuss any further issues you see in its respective repo.  Thanks!

**Pull request closed without merging**